### PR TITLE
Add support for DNS Made Easy

### PR DESCRIPTION
**Pull request by jlduran**
Documentation:

http://www.dnsmadeeasy.com/dynamic-dns/

### CODE REVIEW

File: etc/inc/dyndns.class [hunk 1]
```diff
@@ -1288,6 +1299,33 @@
 						$this->_debug($body);
 					}
 					break;
+				case 'dnsmadeeasy':
+					if (preg_match('/success/i', $data)) {
```
Comment (rbgarga) [inline etc/inc/dyndns.class]: All these preg_match() calls are an overhead. IMO all these if/else blocks should be replace by a switch case block

File: etc/inc/dyndns.class [hunk 2]
```diff
@@ -1288,6 +1299,33 @@
...
+						$status = "phpDynDNS({$this->_dnsHost}): (Success) IP Address Changed Successfully! (".$this->_dnsIP.")";
```
Comment (rbgarga) [inline etc/inc/dyndns.class]: Error messages should have gettext() calls to be able to be translated to other languages

File: etc/inc/dyndns.class (hunk 1 above)
Comment (jlduran) [inline etc/inc/dyndns.class]: I agree that switch cases are more readable, however this seems to be the coding style for this file.

File: etc/inc/dyndns.class (hunk 2 above)
Comment (jlduran) [inline etc/inc/dyndns.class]: In that case, all the other entries should use `gettext()` as well. I am reluctant to change them in this PR.

File: etc/inc/dyndns.class (hunk 1 above)
Comment (jlduran) [inline etc/inc/dyndns.class]: Done!

**Pull request merged**